games/lost-ark/sim_hone.py

```diff
@@ -2,6 +2,7 @@
 import matplotlib.pyplot as plt
 import math
 
+tier4wep = ['<("<)', '^("<)', '(^")>', '(>")>']
```

Comment:
lol what is this

Reply:
it's a pull request dude. c'mon i polluted my python environment to test it

Reply:
@ArcticFaded what do you think?